public/sass/components/_alerts.scss

```diff
@@ -78,6 +79,7 @@
 
 .alert-body {
   flex-grow: 1;
+  width: 50%;
```

Comment:
why is the alert body 50%?

Reply:
Creating a child container- in this case, `Alert-body` -without explicitly setting its width causes the content to overflow.
Because the data is dynamically generated, it's best to set constraints so as to avoid issues like this.

Reply:
I replaced `width` with `min-width` which seems to work pretty fine.
wdyt? @torkelo